inlong-dashboard/src/metas/clusters/common/ClusterDefaultInfo.ts

```diff
@@ -58,7 +59,9 @@ export class ClusterDefaultInfo implements DataWithBackend, RenderRow, RenderLis
         })),
     },
   })
-  @ColumnDecorator()
+  @ColumnDecorator({
+    render: text => text && getClustersType(text),
```

Comment:
It is recommended to use `clusters.find()` directly here, do not export a new method, to avoid the complexity of nesting design model relationships.